data/nlu/nlu.md

```diff
@@ -1256,3 +1255,72 @@
 - cool thanks
 - thanks for forum link, I'll check it out
 - thanks!
+
+## intent:how_to_get_started
+- how do I get started with Rasa
+- how do I use rasa
+- what is the difference between rasa [nlu](product) and rasa [core](product)
+- how to start with rasa
+- how to start with it
+- how to build a bot?
+- how can I build a chatbot
+- what can I do with Rasa?
+- where to start the development of rasa
+- WHAT IS RASA
+- WHAT IS IT
+- so, how do I use rasa?
+- can i just test features without having to deal with your predefined conversation
+- can you guide me know to create knowledge base chatbot
+- how can I learn rasa
+- i want to use rasa to build my chatbot
+- how do i get rasa [core](product)
+- how do I train rasa [nlu](product)
+- how do i learn rasa [core](product)
+- what is rasa [core](product)
+- what is [nlu](product)
+- I want to learn about rasa [core](product)
+- i want to use [nlu](product)
+
+## intent:technical_question
+- what is duckling
+- where to train intents in rasa?
+- duckling
+- dicklung
+- what is duckling
+- i need a help to integrate rasa with a messenger
+- what infrastructure is required to run a bot?
+- can you help me with this problem
+- help me wih the installation
+- having trouble with rasa installation
+- i am having trouble setting this up
+- how about interactive learning
+- interactive learning?
+- technical side of things?
+- does rasa have all the same features as [luis](current_api)
+- is rasa better than [dialogflow](current_api)
+- should I use rasa instead of ibm Watson api
+- Is Rasa better than Dialogflow?
+- db processing
+- how do slots work
+- what components of Rasa are open source
+- how to integrate rasa chatbot with my website
+- how works rasa
+- with exceptions
+- which language is rasa programmed in
+- what technologies did u use to create more mature chatbot?
+- which technology is used to create you
+
+
+## intent:switch
```

Comment:
i think we also need to add some examples of utterances where the user just says "core"/"dialogflow" etc

Reply:
but all this training data needs to be added to the platform anyways, not here